pfSense-pkg-API/files/etc/inc/apicalls.inc

```diff
@@ -19,6 +19,39 @@ api_runtime_allowed();    // Check that our configuration allows this API call t
 session_start();    // Start our session. This is only used for tracking user name
 $pf_ver_num = get_pfsense_version()["program"];    // Pull our full pfSense version
 
+
+function api_system_tables() {
+    # VARIABLES
+    global $err_lib, $config, $api_resp, $client_params;
+    $read_only_action = true;    // Set whether this action requires read only access
+    $req_privs = array("page-all");    // Array of privs allowed
```

Comment:
Thank you for following the format of the other API calls for this. One thing I notice is this will only allow the API call if the user has access to all pages of the webConfigurator. We should also add "page-diagnostics-tables" to this array to accommodate users who have specific access to the Diagnostics > Tables page.

Reply:
Updating and resubmitting shortly